src/board/board.spec.ts

```diff
@@ -0,0 +1,19 @@
+import { Tile } from "../tile/tile";
+import {renderLine, Board, renderBoard, NullableTile} from "./board"
+
+describe("renderLine", () => {
+    it("Testing a line with both undefined and Tile values", () => {
```

Comment:
the "it" function was named like that so your tests become more readable. The test title must be a description of a behavior, e.g. expected output for a given output. In this case, it should be:

```
it('renders each symbol of the board', () =>
```

Alternatively, you can use a Given/When/Then syntax

https://markus.oberlehner.net/blog/naming-your-unit-tests-it-should-vs-given-when-then/

Reply:
Fixed this to "it should" syntax. I will use given / when / then syntax if I see relevant tests later.